dispers/src/jvmMain/kotlin/dev/schlaubi/lavakord/dispers/DispersLavaKord.kt

```diff
@@ -0,0 +1,151 @@
+package dev.schlaubi.lavakord.dispers
+
+import dev.bitflow.dispers.client.DispersClient
+import dev.bitflow.dispers.client.amqp.*
+import dev.kord.common.entity.DiscordChannel
+import dev.kord.common.entity.DiscordGuild
+import dev.kord.common.entity.DiscordVoiceState
+import dev.kord.common.entity.Snowflake
+import dev.kord.gateway.Reconnect
+import dev.kord.gateway.UpdateVoiceStatus
+import dev.kord.gateway.VoiceServerUpdate
+import dev.kord.gateway.VoiceStateUpdate
+import dev.schlaubi.lavakord.LavaKordOptions
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import dev.schlaubi.lavakord.audio.DiscordVoiceServerUpdateData
+import dev.schlaubi.lavakord.audio.Link
+import dev.schlaubi.lavakord.audio.Node
+import dev.schlaubi.lavakord.audio.internal.AbstractLavakord
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.flow.*
+import org.slf4j.LoggerFactory
+import kotlin.coroutines.CoroutineContext
+
+internal class DispersLavaKord(
+    private val client: DispersClient,
+    userId: ULong,
+    shardsTotal: Int,
+    options: LavaKordOptions
+) : AbstractLavakord(userId, shardsTotal, options) {
+
+    override val coroutineContext: CoroutineContext = Dispatchers.IO + SupervisorJob()
+
+    init {
+
+
```

Comment:
Why are there two blank lines?

Reply:
I honestly am not sure, I can fix that.

Reply:
Is this a formatter issue?

Reply:
Probably more my stupidity. I will fix that in the next commit.